src/inputs/Button/Button.tsx

```diff
@@ -48,7 +48,11 @@ export const Button = ({
   return (
     <button
       type="button"
-      onClick={props.onClick}
+      onClick={() => {
+        if (disabled) return;
+        if (isLoading) return;
+        props.onClick()
+      }}
       disabled={disabled}
       variant={"primary"}
```

Comment:
**issue:** I didn't notice it before, but we shouldn't pass `variant` prop to HTML element, because it does not exist. It won't change anything in our element